src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/pool-list/pool-list.component.html

```diff
@@ -18,7 +18,10 @@
 <ng-template #healthTmpl
              let-row="data.row"
              let-value="data.value">
-  <span [ngClass]="row.health_color | mirrorHealthColor">{{ value }}</span>
+  <cds-tag [size]="'sm'"
```

Comment:
```suggestion
  <cds-tag
```
Let's use the default `md` size, what do you think?

Reply:
Done